efi/meson.build

```diff
@@ -20,7 +20,7 @@ endif
 
 efi_libdir = get_option('efi-libdir')
 if efi_libdir == ''
-  cmd = 'cd /usr/lib/$(@0@ -print-multi-os-directory) && pwd'.format(efi_cc)
+  cmd = 'cd /usr/lib/$(@0@ -print-multi-os-directory) && pwd'.format('gcc')
```

Comment:
at least for me this fails with my default CC command (`ccache cc`) if I keep it `efi_cc`.  Any other ideas welcome @rossburton.